### ISSUE TITLE
Any recommendation to install notro fbfft into old Linux 2.6.3x kernel ?

### ISSUE THREAD

**Issue by alexo68** [notro/fbtft#276]
I am work with Freescale iMX233 board and want connect small LCD based on ILI9341 controller. For some reason I must use old Linux 2.6.35 kernel in this project. In documentation described how to add it to Linux 3.5 kernel and "before 3.5". Does it included also old 2.6.3x kernel or some additional steps is required?

**Comment by notro** [comment]
It's before and after **3.15**, when the fbdev directory was added, not 3.5.

I haven't heard of anyone using it on 2.x. When I started this project I was on 3.6.
If I remember correctly, fbtft has been used on 3.2.
With the [use managed gpio requests](https://github.com/notro/fbtft/commit/9644cef2ccd50c727ccb4562b2209a10217c0447) commit, devm_gpio_request_one is used, which came with 3.5.
So you would have to try before that, and probably before the commit before that as well (devm_kzalloc).
